locales/en/recommendations.ftl

```diff
@@ -80,10 +80,9 @@ rec-ip-non-us =
 
 # Recommendation subhead
 rec-moz-vpn-cta = Try { -brand-mozilla-vpn }
-rec-moz-vpn =
-  Your Internet Protocol address (IP address) can reveal your location and internet service provider. A service like
-  { -brand-fpn } hides your IP address and location for { -brand-name } and a service like { -brand-mozilla-vpn } hides your
-  IP address and location for your entire device.
+rec-moz-vpn-update =
```

Comment:
Updated string ID for new translation: 
`rec-moz-vpn` is now `rec-moz-vpn-update`